mcmc_utils.py

```diff
@@ -127,6 +128,11 @@ def fracWithin(pdf, val):
 
 
 def thumbPlot(chain, labels, **kwargs):
+    #TODO:
```

Comment:
I don't think this comment should be in thumbPlot, nor should any changes be made to that function. 

```thumbPlot``` is, and probably should remain, a simple routine that plots anything passed into it. If you wanted to sample the chain first, or select only certain parameters to plot, I think that should be done by modifying the arguments to this function, not this function itself.

BTW - if you want to sample only every ```N``` points from a chain file look at the ```thin``` parameter for routines like ```mcmc_utils.read_chain```

Reply:
I removed this comment.